tests/RobotFramework/tests/tedge_agent/workflows/long-running-command-v1.toml

```diff
@@ -7,7 +7,7 @@ on_success = "executing"
 [executing]
 script = "sleep ${.payload.duration}"
 on_success = "logging"
-on_error = "logging"
+on_kill = "executing"
```

Comment:
The intent of looping back to `executing` state is to resume the operation from here after the agent restart?

Reply:
The intent is to be sure the command is not finalized *before* the agent stops.

Currently, the actors are stopped in a random order, and I observed a run where the script was killed before the workflow engine, the latter being fast enough to move the operation forward before being stopped.